docs/topics/mta-web-seeding-instances.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * docs/rules-development-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-seeding-instances_{context}"]
+= Seeding instances
+
+If you are a project architect you can configure the instances key parameters in the Controls window, before migration. The parameters can be added and edited as needed.
+The following parameters define applications, individuals, teams, verticals or areas within an organization affected or participating in the migration:
+
+* Stakeholders
+* Stakeholder groups
+* Job functions
+* Business services
+* Tag types
+* Tags
+
+You can create and configure an instance in any order. However, the suggested order below is the most efficient for creating stakeholders and tags.
+
+Stakeholders:
+
+. Create stakeholder groups
+. Create job functions
+. Create stakeholders
+
+Tags:
+
+. Create tag types
+. Create tags
+
+Stakeholders and defined by:
+
+* Email
+* Name
+* Job function
+* Stakeholder groups
+
+== Creating a new stakeholder group
+
+You can follow these steps to create a new stakeholder group. There are no default stakeholder groups defined.
+
+.Procedure
+
+. In *Developer* view, click *Controls*.
+. Click *Stakeholder groups*.
+. Click *Create new*.
+. Enter the following information:
+** Name
+** Description
+** Member(s)
+. Click *Create*.
+
+== Creating a new job function
+
+{ProductName} ({ProductShortName}) uses the job function attribute to classify stakeholders and provides a list of default values that can be expanded.
+
+You can follow these steps to create a new job function, which is not in the default list.
```

Comment:
@CarmiWisemon 
Does this really mean that the user can add DEFAULT values? Probably not. Probably, it just means that new job functions can be created. 
I would suggest replacing the two paragraphs with this: 
"{ProductName} ({ProductShortName}) uses the job function attribute to classify stakeholders and provides a list of default values. You can create new job functions by following the procedure below.

Reply:
@CarmiWisemon 
Why do you want to explain that the new job function is not in the default list?  'New' already means that it is not default. Also, it should be 'that', not 'which' and without the comma.